### PR TITLE
Use Dataverse Etag support for version control trigger on web extension

### PR DESCRIPTION
**Pull request by tyaginidhi**
Using Etag support in Dataverse allows to not accidentally over-write the entity data already updated on Dataverse side. 

VscodeWebExtensionEtagSupport

Works for new and old data model

### CODE REVIEW

**Comment by BidishaMS** [comment]
Can you please raise small PRs. I feel we could have done incremental changes. It helps in reviewing the PR also

File: package.json [hunk 1]
```diff
@@ -384,6 +384,11 @@
             "PowerPlatform PCF Debugger for VS Code will use Microsoft Edge Dev version",
             "PowerPlatform PCF Debugger for VS Code will use Microsoft Edge Canary version"
```
Comment (BidishaMS) [inline package.json:385]: Can you please raise small PRs. I feel we could have done incremental changes. It helps in reviewing the PR also

Comment (tyaginidhi) [reply package.json:385]: This one will be harder to de-tangle - BUT i am keeping corner cases coverage, telemetry and further test updates in a separate PR.

**Review by BidishaMS — APPROVED** [review]
